src/renderer/screens/account/AccountHeaderActions.js

```diff
@@ -102,7 +101,16 @@ const AccountHeaderActions = ({ account, parentAccount, openModal, t }: Props) =
   const availableOnCompound = useCompoundAccountEnabled(account, parentAccount);
 
   const availableOnBuy = isCurrencySupported("BUY", currency);
-  const availableOnSwap = useSelector(swapSelectableCurrenciesSelector);
+
+  const { providers } = useSwapProviders();
+  const storedProviders = useSelector(providersSelector);
+  const availableOnSwap =
+    (providers || storedProviders) &&
+    !!(providers || storedProviders).find(({ pairs }) => {
+      console.log({ pairs });
```

Comment:
Is the `console.log({ pairs });` expected / needed?

```suggestion
```

Reply:
Good catch thanks! I let that slip out